tools/cloud_functions/gcs_event_based_ingest/gcs_ocn_bq_ingest/common/ordering.py

```diff
@@ -221,7 +319,21 @@ def handle_backlog(
             start_backfill_subscriber_if_not_running(gcs_client, bkt,
                                                      table_prefix)
             return True  # we are re-triggering a new backlog subscriber
-    utils.handle_bq_lock(gcs_client, lock_blob, None)
+    table = None
+    # Get table from lock blob
+    lock_contents_str = utils.read_gcs_file_if_exists(
+        gcs_client, f"gs://{bkt.name}/{lock_blob.name}")
+    if lock_contents_str:
+        lock_contents: Dict = json.loads(lock_contents_str)
+        if lock_contents:
+            print(
+                json.dumps(
+                    dict(message=f"View lock contents in jsonPayload for"
+                         f" gs://{bkt.name}/{lock_blob.name}",
+                         lock_contents=lock_contents)))
```

Comment:
Syntax here is similar to that of the functions in the logging.py module. Would it not be slightly more consistent if we had something in the logging.py to manage this? In the same manner as in [line 82](- https://github.com/danieldeleo/bigquery-utils/blob/bq_ingest_updates/tools/cloud_functions/gcs_event_based_ingest/gcs_ocn_bq_ingest/common/ordering.py#L82-L83) and [232](- https://github.com/danieldeleo/bigquery-utils/blob/bq_ingest_updates/tools/cloud_functions/gcs_event_based_ingest/gcs_ocn_bq_ingest/common/ordering.py#L232-L234)

Reply:
The logging.py holds helper functions for standardized logging that can be used for log filters/routers. This particular instance of a print/log statement is a one-off which is more for debugging than filtering/routing. I do agree we should manage as much in logging.py as possible but I'd prefer to not created a helper function for just this one print statement.

Reply:
Understood, thanks for the explanation. 

I believe the same print statement is also used [here](https://github.com/danieldeleo/bigquery-utils/blob/bq_ingest_updates/tools/cloud_functions/gcs_event_based_ingest/gcs_ocn_bq_ingest/common/ordering.py#L122)

Anyhow if think we are ok with leaving these print statements as they are. The filters/routers vs debugging explanations makes enough sense.